transport/http/outbound.go

```diff
@@ -300,6 +291,33 @@ func (o *Outbound) callWithPeer(
 		return nil, yarpcerrors.Newf(yarpcerrors.CodeUnknown, "unknown error from http client: %s", err.Error())
 	}
 
+	return response, nil
+}
+
+func (o *Outbound) callWithPeer(
+	ctx context.Context,
+	treq *transport.Request,
+	start time.Time,
+	ttl time.Duration,
+	p *httpPeer,
+) (*transport.Response, error) {
+	req, err := o.createRequest(p, treq)
+	if err != nil {
+		return nil, err
+	}
+
+	req.Header = applicationHeaders.ToHTTPHeaders(treq.Headers, nil)
+	ctx, req, span, err := o.withOpentracingSpan(ctx, req, treq, start)
+	if err != nil {
```

Comment:
This is probably unreachable. We can leave a comment if that’s the case, instead of getting coverage.